src/features/rbac/constants/roleAccessRights.ts

```diff
@@ -33,6 +34,7 @@ export const roleAccessRights: Record<MembershipRole, Scope[]> = {
     "scores:CUD",
     "project:delete",
     "datasets:CUD",
+    "project:transfer",
```

Comment:
would it not make more sense to have 'membership:delete' and 'membership:create' rights here?

Reply:
I removed the role, the owner can transfer their own project